pkg/config/configsections/common.go

```diff
@@ -39,6 +39,12 @@ type Operator struct {
 	// Subscription name is required field, Name of used subscription.
 	SubscriptionName string `yaml:"subscriptionName" json:"subscriptionName"`
 
+	// BundleImage is the URL referencing the bundle image
+	BundleImage string `yaml:"bundleImage" json:"bundleImage"`
```

Comment:
Created a PR (on your PR/ main) adding saving the whole initial runtime environment, including this structure to the claims.jon file. It also includes some linter fixes.
https://github.com/ramperher/test-network-function/pull/1

Reply:
thanks David! I've added your changes, I'll start checking how this change works by running tnf on DCI, checking if we have the desired output. I'll keep you all posted.